starknet_devnet/devnet_config.py

```diff
@@ -353,6 +359,7 @@ def __init__(self, args: argparse.Namespace = None):
         self.hide_predeployed_accounts = self.args.hide_predeployed_accounts
         self.fork_network = self.args.fork_network
         self.fork_block = self.args.fork_block
+        self.fork_retries = self.args.fork_retries
```

Comment:
Probably no need to introduce this here since it's not used anyhwere else in the code other than `devnet_config.py`. And the property name is wrong: `fork_retries` vs `fork_retry`

Reply:
OK. I'll remove it